### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chipironcin**
Update README.md with quick deploy information using Caddy Docker container and add a bit of structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,24 @@ Hello from inside the Matrix.
...
+git clone https://github.com/emilyxxie/matrix_vision.git
+docker run -v $(pwd)/matrix_vision:/srv -p 2015:2015 abiosoft/caddy
+```
+Then open your browser on `localhost:2015`.
+
+## Tips
 
-![alt tag](matrix_preview_2.gif)
+* Avoid being backlit for best results
+* Doesn't work on Firefox Quantum because `navigator.getUserMedia()` is obsolete (see [Issue#2](https://github.com/emilyxxie/matrix_vision/issues/2))
```
Comment (emilyxxie) [inline README.md]: Just merged in this issue––let's remove this line now!

**Comment by chipironcin** [comment]
Done 😄

**Comment by chipironcin** [comment]
ping

**Comment by emilyxxie** [comment]
Just merged in! Sorry, I haven't checked this repo in forever. :-)

**Comment by chipironcin** [comment]
Thank you! Very cool repo BTW ;)

**Pull request merged**